Content: Unfettered campaign Act 1 (#6416)
endless-sky/endless-sky: data/hai/unfettered 1 alphas.txt

		has "midas: tracker relay"
	on offer
		conversation
			`As you leave your ship, an Unfettered Chief approaches you. After some time you finally recognize Sookootow, the one you helped earlier with moving people and medical supplies. You have not yet seen him so worried, nor moving at such speed.`
			`	"<first>! I asked to be notified of your arrival as soon as possible, as we need your services. A fleet of Hai needs your immediate assistance escorting them through Hai space, from <stopovers>. They may not be Unfettered, but their hearts beat true still. I would blame myself if anything was to happen to them, and I could have done anything to prevent it."`
			choice
				`	"Why do you need my help specifically?"`
				`	"What are you even doing here?"`
					goto purposehere
				`	"I'm busy right now, sorry."`
					goto busy
			`	"It is not that I do not trust the Hai not to fire on their own people, but a misunderstanding happens quickly. Someone of your expertise would be helpful, and I cannot simply trust any Unfettered captain with this, as they could easily be provoked into making a fatal mistake. I hope, though, that you will be able to make a rational decision.`

> 			`	He gets closer to you, and looks around to make sure nobody listens, except for one Unfettered, moving as his shadow, that you assume to be his bodygard. "I also need your presence because I want someone that is quick to respond to danger, and with the right tools. Hearing of your duel with a fellow Unfettered captain, you strike me as such a person. We are worried that another party may interfere with our plans."`

Reviewer: ```suggestion
			`	He gets closer to you, and looks around to make sure nobody listens, except for one Unfettered, moving as his shadow, that you assume to be his bodyguard. "I also need your presence because I want someone that is quick to respond to danger, and with the right tools. Hearing of your duel with a fellow Unfettered captain, you strike me as such a person. We are worried that another party may interfere with our plans."`
```

			choice
				`	(Do as told.)`
				`	(Try to defend yourself.)`
					goto defend
			`	She shoots at your shoulder, making you feel dizzy for a while.`
				goto recovered
			label defend
			`	To her surprise, you try to dodge the shot, but all you manage to do is to get yourself stunned by the consecutive ones, all apparently aiming at your shoulder. You almost lose consciousness.`
			label recovered
			action
				clear "midas: tracker relay"
				"midas: wronged" ++
			`	Once you recovered, she explains. "Someone had placed a tracker on your shoulder. A very nasty kind of tracker, wired with communications capabilities, and I did not want to find out what else it could do before shooting it. Sorry if I took you by surprise, Captain, but you did well to warn us, and the governor will reward you justly for it once we arrive at the destination."`
			label leave
			branch otherships
				"ships: Space Liner" + "ships: Transport" + "ships: Light Freighter" + "ships: Heavy Freighter" + "ships: Utility" + "ships: Interceptor" + "ships: Light Warship" + "ships: Medium Warship" + "ships: Heavy Warship" >= 1

> 			`	The Hai bodygards now leave your ship, reassured that you are not of any threat for their fleet.`

Reviewer: ```suggestion
			`	The Hai bodyguards now leave your ship, reassured that you are not of any threat for their fleet.`
```

				goto admiral
			label otherships

> 			`	The Hai bodygards now leave your ship, and proceed to check the other ships of your fleet, before reporting a green light.`

Reviewer: ```suggestion
			`	The Hai bodyguards now leave your ship, and proceed to check the other ships of your fleet, before reporting a green light.`
```
Another one???

				to display
					not "midas: wronged"
			`	"Do not think we have forgotten of your recent deeds, <first> <last>. But you have proved your worth today."`
				to display
					has "midas: wronged"
			`	You receive a final, private message from one of the ships. "A shame you brought such a fleet with you, we could have had so much fun otherwise... Ra told us you know how to fight."`
				to display
					"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 3 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 + "ships: Superheavy" * 18 > 24
				goto leaving
			choice
				`	"Thanks for the assistance."`
					goto leaving
				`	"Wait! I challenge you to a duel for those weapons!"`
			`	They seem surprised, but not disappointed in the slightest, "A challenge, interesting," one says, "We would have much to learn from each others," the other one add. "But what do we have to gain? Our shredders are worth many millions, so I think it would be fair you repayed their value to us, should you lose."`
			choice
				`	"Seems like a fair deal to me."`

> 				`	"Nevermind, I'll let you be on your way."`

Reviewer: ```suggestion
				`	"Never mind, I'll let you be on your way."`
```

				outfit "Gold Bars" -10
			`	"That would be acceptable, yes." They take hold of the Gold Bars and verify them with a small device before continuing. "Here is a set of coordinates." They do not give any more details.`
			label "okay boy!"
			action
				set "unfettered: it's the Alphas!"
			`The coordinates lead you to an enclave on the fringes of town. It's an outdoor beer garden, and the regulars eye you with suspicion. You order a drink and grab a table. No one approaches you, so you just look around and watch people come and go. The only thing that catches your eye is a mural painted on the wall of the building opposite. It reminds you of ancient illustrations you've seen of a knight fighting a dragon. But in this version, the knight is wearing light battle armor and hovering with some kind of jet pack. His lance is made of a stream of shots from a Hai pulse rifle. You can't see wings on the dragon, but it has a weapon firing bright red lines of what must be plasma.`
			`	You order another drink and look at the mural in more detail. You nearly choke on your drink when you realize that the exposed face and hands of the knight are green.`
				decline
			label "dead, boy!"

> 			`As soon as you set foot on the planet, a team of mercenaries comes out of the shadows, equippped with the same kind of armor Si-Ra had. Their weapons may be holstered, but you can feel the tension in the air. Their leader speaks up. "We know your query. Consider yourself lucky to not be killed on the spot. We do not guarantee that will not happen, should you land again."`

Reviewer: ```suggestion
			`As soon as you set foot on the planet, a team of mercenaries comes out of the shadows, equipped with the same kind of armor Si-Ra had. Their weapons may be holstered, but you can feel the tension in the air. Their leader speaks up. "We know your query. Consider yourself lucky to not be killed on the spot. We do not guarantee that will not happen, should you land again."`
```

				die
			label hint
			`	Some of them already aim for you with remarkable speed, ready to shoot, but what you suspect to be their leader makes them lower their weapons again with a gesture of their hand. "The ones you seek are a common enemy, so I will tell you this: look far, human, and you may find them in the eye of a storm that has been raging for centuries." They then disappear as quickly as they had come.`

mission "<uhai> 9: Alpha Quest"
	name `Find the Auxiliary`
	description `Go to <destination> to learn where you can find the Alpha ship.`
	source "Darkcloak"
	destination "Firelode"
	to offer
		not "unfettered: destroyed auxiliary"
		has "<uhai> 7: Stolen Cargo: done"
		has "event: unfettered: analyzed the scans"
		has "unfettered: it's the Alphas!"
		has "Unfettered Jump Drive 3: offered"
	on offer
		conversation
			`As you enter the spaceport, you see the council gathered in a stadium looking similar to the one on Firelode, although it is larger. It seems like you will finally be able to get some answers, if you would like to see them now, that is.`
			choice
				`	(Enter the stadium and go see them.)`
				`	(I don't have time right now.)`
					defer

> 			`	Seeing you approach, one of the leaders steps forward, and you recognize Sookootow. "It seems you are an enemy of the unfettered humans. We cannot risk a disruption in our supply of jump drives. You may have provided some to us, but so did they," He says, speaking almost theatrically, with big geastures, "We have a task to ask of you. Should you accept, you would be able to join us - still not officially for now, but all good things come in time." You assume that by unfettered humans he means the Alphas.`

Reviewer: ```suggestion
			`	Seeing you approach, one of the leaders steps forward, and you recognize Sookootow. "It seems you are an enemy of the unfettered humans. We cannot risk a disruption in our supply of jump drives. You may have provided some to us, but so did they," He says, speaking almost theatrically, with big gestures, "We have a task to ask of you. Should you accept, you would be able to join us - still not officially for now, but all good things come in time." You assume that by unfettered humans he means the Alphas.`
```

	on enter "Persitar"
		dialog `This doesn't look like a good hiding spot...`
	on enter "Ferukistek"
		dialog `There are traces here indeed. You spot one system with no hyperspace links north-east of here. It seems like a logical place for the Alphas to go, considering the drones do not appear to have jump drives.`
	on enter "Host"
		dialog
			`You arrive in the system expecting to find a vault and the ship repairing, but it looks more like... It's a trap!`
			`	The Auxiliary from earlier is indeed here, but it's not damaged at all, and it looks like it's waiting for you with quite a few drones, more than last time.`
			`	Whatever the case, it probably has the jump drives on board or on the nearby planet. You should first capture the ship, as asked by the Unfettered, and then inspect the planet. Strangely, you do not find a single trace of the particle trace you have been following so far in this system.`
	npc capture
		government "Cerberus"
		system "Host"
		personality heroic nemesis staying vindictive target
		ship "Auxiliary (Alpha)" "Settler"
		conversation
			action
				set "unfettered: captured the auxiliary"

> 			`While capturing the ship, you noticed that only very few of the crew were actually Alphas. What little were Alphas seem to have disappeared in the middle of the fight. They probably used an escape pod to get away unoticed, or maybe they were never truly there in the first place. Whatever the case, you cannot find any trace of them. The crew was composed of a mix of mercenaries and indoctrinated pirates that would obey only to 'the Masters' and did not reveal anything useful.`

Reviewer: ```suggestion
			`While capturing the ship, you noticed that only very few of the crew were actually Alphas. What little were Alphas seem to have disappeared in the middle of the fight. They probably used an escape pod to get away unnoticed, or maybe they were never truly there in the first place. Whatever the case, you cannot find any trace of them. The crew was composed of a mix of mercenaries and indoctrinated pirates that would obey only to 'the Masters' and did not reveal anything useful.`
```

			`	When you arrive they are discussing among themselves again, and the warlord from earlier speaks as he sees you, seemingly with a heavy burden. "Now that you officially work for us, you will receive a daily share of 1000 credits. We will not make you wear our colors yet, because it could cause problems with the Hai and those parrots that stole our worlds up north. We wish for you to be free to travel in their territory." They then make the necessary preparations to transfer the Korath out of your ship, after stabilizing them as much as possible.`
			branch logs
				has "<uhai> 9: Alpha Base: done"
			# The Alpha logs self-destruct if you didn't make a copy of them earlier.

> 			`	Having nothing more to show the Unfettered, you leave. This leaves you wondering if there was something hidden in the Auxiliary, but after a short inspection you summize that if there was, it is no longer there now.`

Reviewer: ```suggestion
			`	Having nothing more to show the Unfettered, you leave. This leaves you wondering if there was something hidden in the Auxiliary, but after a short inspection you surmise that if there was, it is no longer there now.`
```
Alternatively, summarize.

outfitter "Unfettered Alpha Trade"
	"Tripulse Shredder"
	"Ion Cannon"
	"Hai Tracker Pod"
	"Hai Tracker"
	"Pulse Turret"
	"Value Detector"
	`"Bufaer" Atomic Thruster`
	`"Bufaer" Atomic Steering`
	"Pulse Rifle"
	"Hai Valley Batteries"
	"Boulder Reactor"
	"Hai Octopus Jammer"
	"Wanderer Ramscoop"
	"Sunbeam"
	"Thunderhead Launcher"
	"Thunderhead Missile"
	"Bright Cloud Shielding"
	"Small Biochemical Cell"
	"Yellow Sun Reactor"
	"Type 3 Radiant Thruster"
	"Type 3 Radiant Steering"

mission "Unfettered: Alpha Supplies"
	invisible
	minor
	repeat
	landing
	source "Warfeed"
	deadline 3
	to offer
		not "<uhai> 9: Alpha Quest: offered"
		not "unfettered: destroyed auxiliary"
		not "Unfettered: Alpha Supplies: active"
		"reputation: Hai (Unfettered)" >= 0
		or
			day <= 2
			and
				"day" >= 14
				"day" <= 15
		random < 40 - "alpha: worry" * 5
	on enter "Hi Yahr"
		"unfettered: saw auxiliary" ++
		conversation
			branch enough
				"alpha: worry" >= 2
			branch leaving
				or
					day == 2
					day == 15
			`As you launch, the local authorities hail you, "An important guest is arriving, you should not be here when they do. Get out, fast."`

> 			`	They insist: "Whatever you do, do not destroy that incoming ship."`

Author: idk about this, I prefer the way it sounds rn

			action
				"reputation: Hai (Unfettered)" -= 1000
	npc kill
		government "Alpha"
		to spawn
			or
				"day" == 1
				"day" == 14
		system "Hi Yahr"
		personality entering uninterested fleeing timid
		"cargo settings"
			cargo 4
			commodities "Military" "Electronics"
			outfitters "Korath Drones Alpha Loot"
		ship "Auxiliary (Alpha)" "Settler"
		on kill
			set "unfettered: destroyed auxiliary"
			conversation "unfettered outcast"
		on board
			"alpha: worry" ++
	npc kill
		government "Alpha"
		personality launching uninterested fleeing timid
		system "Hi Yahr"
		to spawn
			or
				"day" == 3
				"day" == 16
		"cargo settings"
			cargo 6
			commodities "Metal" "Heavy Metals"
			outfitters "Unfettered Alpha Trade"
		ship "Auxiliary (Alpha)" "Settler"
		on kill
			set "unfettered: destroyed auxiliary"
			conversation "unfettered outcast"
		on board
			"alpha: worry" ++
	npc
		government "Alpha"
		to spawn
			or
				"alpha: worry" == 2
				"alpha: worry" == 3
		personality vindictive heroic entering coward

> 		ship "Met Par Tek 53" 2

Author: yeah they dont get out of there, they're only sacrificial

				goto beforeugo
			label price
			`	"Yes, the price part needs to be addressed... But their strength is still something to be respected. The best advantage is that the fighters enjoy the ride and do not need independent hyperdrive systems." She seems a bit disappointed about your opinions on fighters, but what you said does not seem to change any big plans for her.`
				goto beforeugo
			label stories
			`	"Indeed, so have I, and for once the stories were not exaggerated compared to the facts! I've heard stories ever since the Carrier was first seen by our people. We compared it to a space station."`
			label beforeugo
			`	"You should know that examining these specs will take me some time, so it could be a while until I employ you again," she says over her head whilst leaving.`
			choice
				`	"Wait, what are the 'unfettered humans'?"`
					to display
						not "unfettered: it's the Alphas!"
				`	(I'll find it out on my own.)`
					decline

> 			`	She stops on her track, before turning around partially to answer, "We use 'unfettered' to describe groups that fight for what they believe in. That includes the anarchists organizations you refer to as pirates, but not only." She is clearly unwilling to tell you more than this and goes back on her way.`

Author: the decline at the end of every mission conversation is implicit

> 			`	She stops on her track, before turning around partially to answer, "We use 'unfettered' to describe groups that fight for what they believe in. That includes the anarchists organizations you refer to as pirates, but not only." She is clearly unwilling to tell you more than this and goes back on her way.`

Reviewer 2: Was not aware of that, this is resolved.

				`	"I'll go there but first I must get a proper ship for this."`
					goto beback
			label dewit
			`	"Good! The ships are mostly carrying supplies and passengers, so you would do well to stay with them whatever happens, as they lack the means to defend themselves properly. If I understand correctly, a small Hai fleet will accompany you too, but they may prove insufficient to ensure their safety. We never know what could happen, even in the safety of Hai space..."`
			`	He takes you aside and murmurs, "Do not repeat this as to not alarm the general population, but there are reasons to believe this was no natural event. Some prominent Hai will be part of the medical teams, for reasons you need not know, and it would be a disaster if something was to happen to them."`
				accept
			label beback
			`	"If you need another ship you should indeed get one, but I cannot allow you to have more time than until <date>, so you must simply get a ship on the way, or before landing on <stopovers>. Try your best, <last>, your reputation with us depends on it!"`
			choice
				`	"I'll find a way to land there with a battle-ready ship."`
					goto dewit
	on enter "Due Yoot"

> 		dialog

Author: I like the way dialogs look more, especially in this space context

> 		dialog

Reviewer 2: Yes, but you have text too long to fit into a dialog box.

> 		dialog

Author: The size is dynamic, iirc it fits

			label otherships
			`	The Hai bodyguards now leave your ship, and proceed to check the other ships of your fleet, before reporting a green light.`
			label admiral
			`	An hour later, the admiral hails the fleet, "All ships are now ready to leave. All that is left is for me to wish you all safe trip. May the leaves flow our way." You depart with the fleet.`
				flee

event "unfettered: shipment lost"

outfit "Building Materials"
	category "Special"
	thumbnail "outfit/unknown"
	cost 1000
	plural "Building Materials"
	"mass" 1
	installable -1
	description "These appear to be simple building materials, probably meant for the construction of houses. It seems weird to go trough all of this trouble just for these, but it might just be because of the Unfettered's pride."

ship "Bactrian" "Bactrian (Cerberus)"
	add attributes
		"outfit space" 17
		"weapon capacity" 20
	outfits
		"Sidewinder Missile Launcher" 2
		"Sidewinder Missile" 90
		"Torpedo Launcher" 2
		"Torpedo" 60
		"Heavy Laser Turret" 4
		"Heavy Anti-Missile Turret" 2
		"Fusion Reactor"
		"Geode Reactor"
		"Hai Gorge Batteries"
		"D67-TM Shield Generator"
		"Hai Williwaw Cooling" 3
		"Ramscoop"
		"Laser Rifle" 20

> 		"Fragmentation Grenades" 20

Reviewer 2: It’s a pirate ship, so,
```suggestion
		"Fragmentation Grenades" 20
		"Nerve Gas" 200
```
Maybe a bit less, but still.

Author: An alpha ship, even
But you're supposed to be able to capture it (except if this is the other variant)

Author: oh wait this is the Cerberus ship not the Alpha ship, good point then but no more than 15 caus its pricy

Reviewer 2: > An Alpha ship, even
> But you're supposed to be able to capture it (except if this is the other variant)

Nerve Gas doesn’t have a great defense. 
They maybe shouldn’t have any because no other ships in the game data have it, but…

Author: I got another ship that has some (of the rival faction though)

Reviewer 2: Your choice on whether it should use.

Reviewer 2: Resolved in 3800bed.

		"Outfits Expansion" 2
		`"Bufaer" Atomic Thruster`
		`"Bufaer" Atomic Steering`
		"Hyperdrive"
	cargo
		outfits
			"Building Materials" 190
		commodities
			"Equipment" 100
			"Electronics" 100
			"Food" 100

mission "<uhai> 7: Stolen Cargo Notification"
	minor
	landing
	source
		attributes "unfettered"
	to offer
		has "event: unfettered: shipment lost"
		not "<uhai> 7: Stolen Cargo: active"
		not "<uhai> 7: Stolen Cargo: done"
		random < 60
	on offer
		conversation
			`Upon landing you take a look at the job board, almost by habit, and to your surprise you notice a very peculiar job, left by none other than Sookootow, titled "looking for stolen cargo".`
			`	It does not look like an urgent job, so it should still be there the next time you land on an Unfettered world, but you should probably check it out when you have the time.`
				decline

# SPOILER: Cerberus or Midas bring cargo to the Unfettered. Cerberus was on the way, but Midas hired Scar's legion to sneakily take it and bring it to them.
# only that Cerberus were on their tail, and just took down the ship and it's cargo when you arrive. Time to steal it back back back
mission "<uhai> 7: Stolen Cargo"
	job

> 	name `Looking for stolen cargo (Sookootow)`

Author: I dont think it is, I remember looking at it in the job board
but all the same that "looking for" is not needed

> 	name `Looking for stolen cargo (Sookootow)`

Reviewer 2: Then resolve it.

> 	description `Apt captain required to find the pirate vessel that has stolen the materials needed for the development of the urban areas. Bring 142 tons back to <destination>, the ship may be found in north-east or north-west pirate space.`

Reviewer 2: ```suggestion
	description `Apt captain required to find the pirate vessel that has stolen the materials needed for the development of the urban areas. Bring 142 tons back to <destination>. The ship may be found in north-east or north-west pirate space.`
```

	clearance `As soon as you say that you are here on behalf of the Unfettered, you are cleared to land.`
	source
		attributes "unfettered"
	to offer
		has "event: unfettered: shipment lost"
	substitutions
		<abducted> "only wishing to scare us with the possibility of abducting"
		<abducted> "although to be completely frank I have no clue how they managed to abduct"
			has "<uhai> 6 A: Loyal Escort: failed"
	on accept
		conversation
			`	You go to see Sookootow in order to discuss this job in greater details, and easily find him by asking around, everyone seems to know him here. He was having a quite animated conversation with an elder, but he puts a stop to it when he notices you.`
			`	"<last>. I am still saddened by what happened to that governor you had to escort... She used to say that all Hai are together, by heart. I will continue to honor her memories."`
				to display
					has "<uhai> 6 A: Loyal Escort: failed"
			`	"<last>! I was hoping to find you here."`
				to display
					has "<uhai> 6 A: Loyal Escort: done"

> 			`	"I just happen to be in needs of your services. Remember the mission you did earlier, escorting Hai ships? Well that shipment was not the only one that got attacked by pirates, in fact I believe their attack on it may have been nothing more than a distraction; they knew full well they could not take that cargo in the middle of Hai space, <abducted> the governor."`

Reviewer 2: ```suggestion
			`	"I just happen to be in need of your services. Remember the mission you did earlier, escorting Hai ships? Well that shipment was not the only one that got attacked by pirates, in fact I believe their attack on it may have been nothing more than a distraction; they knew full well they could not take that cargo in the middle of Hai space, <abducted> the governor."`
```

			`	"The materials they supply us with are not urgently required, but they are needed in order to develop some of the most distant areas on this world, and maintain our current buildings." He then takes a pad out of his pocket, and shows you what they should look like. These are fairly standard materials for construction purposes, not rare in the slightest.`
			`	"We need at the very least 142 tons of this cargo back, and that is conveniently the amount our Shield Beetles can carry."`
			choice
				`	"Do you have any idea where the stolen cargo could be?"`
					goto cargo
				`	"Why not just buy these materials again?"`
					goto buyagain
			label buyagain
			`	"Whilst the cargo itself has no real value, the trip bringing it here costs significantly more, and we do not want to have to pay for it twice. Then there is also the concern that paying for it again will only result in it getting 'lost' a second time... If there is a problem with our supply chain we need it dealt with first. You must be wondering who took the cargo, well...`
			label cargo

> 			`	"From what I was told, there are 2 main human factions in the area, as well as unfettered humans. The ship always maintains a low profile and poses as a merchant. The syndicate would not mind a bit of smuggling, and the navy could only take it in the unlikely event that their cover was blown and they could not escape.`

Reviewer 2: ```suggestion
			`	"From what I was told, there are two main human factions in the area, as well as the unfettered humans. The ship always maintains a low profile and poses as a merchant. The Syndicate would not mind a bit of smuggling, and the Navy would only take it in the unlikely event that their cover was blown and they could not escape.`
```

			label choose
			choice
				`	"I will try to bring the loot back without attracting too much attention."`
					goto finalmention
				`	"I will find them and bring the loot back, whatever the cost.`
					goto whatevercost
				`	"Why not just hire another pirate?"`
					to display
						not "unfettered: why not hire another question"
					goto whyhire
				`	"Why is the trip so dangerous if the cargo is worthless?"`
					to display
						not "unfettered: worthless question"
			label worthless
			action
				set "unfettered: worthless question"
			`	"Our contact mentioned having to sometimes fight alien invaders, although they did not want to specify which. They also have to pass through the big no man's land that leads to the wormhole afterwards, which has some pirates here and there. However, maybe she exaggerated the risks, in order to increase the cost. We are quite desperate for materials, and thus ready to pay quite a hefty price." There might be more to it than that, but he doesn't seem ready to divulge more information to you.`
				goto choose
			label whyhire
			action
				set "unfettered: why not hire another question"

> 			`	"I do not trust them much, but the real reason is that it was our contact that dealt with these kind of preoccupations - we only provided the founds. Sometimes it is one faction that brings us the cargo, sometimes another, and we do not ask questions. So, what do you say? Can you find that cargo and bring it to us?"`

Reviewer 2: ```suggestion
			`	"I do not trust them much, but the real reason is that it was our contact that dealt with these kind of preoccupations - we only provided the funds. Sometimes it is one faction that brings us the cargo, sometimes another, and we do not ask questions. So, what do you say? Can you find that cargo and bring it to us?"`
```

				goto choose
			label whatevercost
			`	He seems a bit wary of your attitude, "Please, whatever you do, try to avoid needless bloodshed, these pirates are not our enemies. I would hardly recommend bribing those that can be. That said, I expect you to do what you must in order to bring back all 142 tons of materials."`
			label finalmention
			`	"You can either fit all of the cargo on a Shield Beetle, or bring them back in multiple trips, by storing them in an outfitter or on your other ships. And remember to look for it north-east of human space."`
			action
				clear "unfettered: worthless question"
				clear "unfettered: why not hire another question"
			branch nvm
				has "flagship model: Shield Beetle"

> 			`	"You must know of the current food shortages we are facing. Far from me to suggest you should profit from it, but I know one can easily make a lot of money moving cargo around in Aphids with the tribute and aid jobs, especially if they do not risk being attack by us, ha!"`

Reviewer 2: ```suggestion
			`	"You must know of the current food shortages we are facing. Far from me to suggest you should profit from it, but I know one can easily make a lot of money moving cargo around in Aphids with the tribute and aid jobs, especially if they do not risk being attacked by us, ha!"`
```

				to display
					has "unfettered: hunger"
					credits < 32000000
			`	"I would advise you to lead this mission with a Shield Beetle, like a true warrior. You deserve it, and it could prove useful."`
			label nvm

> 			`	"It would also probably be best to limit yourself to say 2 heavy warships so you don't attract too much attention."`

Reviewer 2: ```suggestion
			`	"It would also probably be best to limit yourself to, say, two heavy warships so you don't attract too much attention."`
```

			# no need to accept because this is a job; it was already accepted.
	on enter "Misam"
		conversation
			`As you enter the system, you notice pirates, presenting themselves as part of the Cerberus operations, in orbit around the planet. They could have the cargo you are looking for.`
			choice
				`	"I demand you return the materials to me."`
				`	(I have no need to talk with these scums. I shall take.)`
					goto over
			`	The first mate of the <npc> responds to your hail, "You talking about the cargo we just relieved from that scum posing as a merchant? That's ours, mate, always has been. You must have suffered a serious blow to the head to think otherwise. In fact we were the ones that these were stolen from before we could bring it to port! Well, after we stole them from an actual merchant, but that's irrelevant."`
			choice
				`	"What do you intend to do with this cargo?"`
					goto intention
				`	"If you do not give it to me, I will open fire."`
			`	"If I laid down at every threat I received, the Masters would have replaced me long ago. Not gonna lie, I don't mind bashing you down a bit."`
				goto over
			label intention

> 			`	"That's none of your business, but if it can help you get away... My Masters are coming soon to retrieve the cargo quite a few systems away from here, and if it ain't there I'll be the one they 'retrieve' instead. So forget about it!" He says, before abruptly closing the communication.`

Reviewer 2: ```suggestion
			`	"That's none of your business, but if it can help you get away... my Masters are coming soon to retrieve the cargo quite a few systems away from here, and if it ain't there I'll be the one they 'retrieve' instead. So forget about it!" He says, before abruptly closing the communication.`
```

			label over
			`	They quickly land on the planet, probably in order to avoid a fight, or maybe just to win some time... It looks like you will have to land too in order to retrieve the cargo.`
	on enter "Alcyone" "Durax"

> 		dialog "When entering the system you spot small traces of the same kind of cargo Sookootov asked you to bring back. You cannot be far."

Reviewer 2: ```suggestion
		dialog "When entering the system you spot small traces of the same kind of cargo Sookootow asked you to bring back. You cannot be far."
```

			label leave
			`	"She of the Void does not forget. Nemo Me Impune Lacessit, <last>. Do not presume to attempt to betray us again or there will be consequences."`
				to display
					not "midas: tracker relay"
				goto leaving
			`	"The Shadow of Mintaka sends her regards, <first> <last>. We look forward to dancing with you again."`
				to display
					not "midas: wronged"
			`	"Do not think we have forgotten of your recent deeds, <first> <last>. But you have proved your worth today."`
				to display
					has "midas: wronged"
			`	You receive a final, private message from one of the ships. "A shame you brought such a fleet with you, we could have had so much fun otherwise... Ra told us you know how to fight."`
				to display
					"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 3 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 + "ships: Superheavy" * 18 > 24
				goto leaving
			choice
				`	"Thanks for the assistance."`
					goto leaving
				`	"Wait! I challenge you to a duel for those weapons!"`

> 			`	They seem surprised, but not disappointed in the slightest, "A challenge, interesting," one says, "We would have much to learn from each others," the other one add. "But what do we have to gain? Our shredders are worth many millions, so I think it would be fair you repaid their value to us, should you lose."`

Reviewer 2: ```suggestion
			`	They seem surprised, but not disappointed in the slightest, "A challenge, interesting," one says, "We would have much to learn from each other," the other one adds. "But what do we have to gain? Our shredders are worth many millions, so I think it would be fair you repaid their value to us, should you lose."`
```

			choice
				`	"Seems like a fair deal to me."`
				`	"Never mind, I'll let you be on your way."`
					goto leaving
			action
				set "midas: accepted duel"
				event "cerberus revealed"
			branch cleanupdone
				"unfettered: cleaned up cerberus" > 1

> 			`	"First we need to finish off all of these Cerberus pirates remaining. Those dogs only obey the Alphas and cannot be reasoned with. Do not let them live to tell the tale. Once that is done, however" one starts, before the other one continues.`

Reviewer 2: ```suggestion
			`	"First we need to finish off all of these Cerberus pirates remaining. Those dogs only obey the Alphas and cannot be reasoned with. Do not let them live to tell the tale. Once that is done, however," one starts, before the other one continues.`
```

			label cleanupdone

> 			`	"We will let you land and repair before starting the fight. If you want to surrender, just abort the mission of land again after the battle has started, after giving us the symbolic bribe to stop the fight. We do not have the luxury of time!"`

Reviewer 2: ```suggestion
			`	"We will let you land and repair before starting the fight. If you want to surrender, just abort the mission or land again after the battle has started, after giving us the symbolic bribe to stop the fight. We do not have the luxury of time!"`
```

	to fail
		has "unfettered: outcast"
	on fail
		dialog phrase "Essential Unfettered Mission Fail"
	on visit
		dialog `You do not have the materials yet, or your escort carrying them has not arrived. The materials should be looked for in north-east pirate space.`
	on complete
		clear "unfettered: boarded bactrian"
		outfit "Building Materials" -142
		payment 2000000
		log `Helped Sookootow by bringing cargo to help maintain and build houses. This whole story feels weird, with pirates stealing from other pirates, and one may wonder how it is related to the Alphas, and Shadow.`
		conversation
			`You look for Sookootow in the spaceport, and find him talking with Si-Ra, who was about to leave. Seeing you, Si-Ra shows a quick greeting in sign of respect, before leaving.`
			`	The Chief then shakes your hands in true human fashion. "Jiko has just told me of your success<, unfettered title>. Congratulations are in order!"`
			choice
				`	"Thanks. Here is the cargo."`
					goto cargo
				`	"Jiko? I thought his name was Si-Ra."`
			action
				set "unfettered: Jiko true Si-Ra"

> 			`	"He wishes that were his name! When we appoint a warlord, we also give him a name, indicative of his function. He may then wear it proudly, and that name protects him as well. Real names have power, you see... Do you not react when hearing your name, <first>? It identifies you, and a warlord is meant to become different when he earns the title. He must fit the task. To represent that, their names are changed, though they can still use their old names.`

Reviewer 2: ```suggestion
			`	"He wishes that were his name! When we appoint a warlord, we also give him a name, indicative of his function. He may then wear it proudly, and that name protects him as well. Real names have power, you see... do you not react when hearing your name, <first>? It identifies you, and a warlord is meant to become different when he earns the title. He must fit the task. To represent that, their names are changed, though they can still use their old names.`
```

> 			`	"Si-Ra actually means the raging sea, it is a title appointed to the warlords that must stand against enemies, no matter how strong, and never give up or break. They may lose battles, but never the war. As the sea, they can be calm. However, in time of need, they can show sign of the flame burning within them, and let it out." He seems very passionate about it. It occurs to you he probably participated in the creation of some of the warlord titles. "The sea takes all kinds of punishment without real damage, and some think they may tame it, as space... They can't. Those named Si-Ra are the masters of the sea, the masters of space. They never give any information to the enemy when captured, and..."`

Reviewer 2: ```suggestion
			`	"Si-Ra actually means the raging sea, and it is a title appointed to the warlords that must stand against enemies, no matter how strong, and never give up or break. They may lose battles, but never the war. As the sea, they can be calm. However, in time of need, they can show sign of the flame burning within them, and let it out." He seems very passionate about it. It occurs to you he probably participated in the creation of some of the warlord titles. "The sea takes all kinds of punishment without real damage, and some think they may tame it, as space... they can't. Those named Si-Ra are the masters of the sea, the masters of space. They never give any information to the enemy when captured, and..."`
```

				decline
			label cargo
			`	"You well deserve these <payment> for it! I will post jobs in the job board if this event happens again. Unfortunately, I suspect it will. Oh, and if you have any materials still left you can sell them in the outfitter."`

conversation "cerberus reinforcements"
	branch boardedship
		has "unfettered: boarded bactrian"
		has "unfettered: cleaned up cerberus"
	action
		"unfettered: cleaned up cerberus" ++
	`Now that all pirate reinforcements have been taken care of, it is time to board the flagship and take their loot.`
		to display
			"unfettered: cleaned up cerberus" == 2
		decline
	`Part of the pirate reinforcements have been taken care of, but more remain.`
		to display
			"unfettered: cleaned up cerberus" == 1
		decline
	label boardedship
	branch noduel
		not "midas: accepted duel"
	`All pirate reinforcements have been taken care of, and the Sea Dragons signal you they are ready to start the duel whenever you will take off again.`
	label noduel

> 	`All pirate reinforcements have been taken care of, all that is left to do is to bring back the cargo.`

Reviewer 2: ```suggestion
	`All pirate reinforcements have been taken care of, and all that is left to do is to bring back the cargo.`
```

mission "<uhai> 7: Cavalry Arrives"
	landing
	invisible
	source "New Tortuga"
	to offer
		has "<uhai> 7: Stolen Cargo: active"
	on offer
		conversation
			`Realizing you have landed after them, the pirates fire up their engines and take off as fast as they can. They might not have expected you to have the guts to land on such a lawless world. Or maybe they just wanted to delay you... Whatever the case, you quickly order your crew to get back into space.`
				launch
	on enter
		conversation
			"As you enter the system it becomes clear: Cerberus have called for backup! Multiple pirate ships are arriving in the system. But you also see other ships arriving by jump drives. They do not appear to be pirates, maybe the 'contact' has provided help as well?"

> 			`	The 2 mysterious black ships hail you, "Listen carefully. You may have noticed a Korath drone in the area, and they don't like Alpha dogs' presence in our space. If some Korath come to help, do not open fire on them, understood?" The other one then speaks, "We'll let you deal with that bactrian, but I would recommend bribing 'neutral' pirates<, unfettered title>!" They then end the hail, making it clear they will not take no for an answer. They probably are allies of the 'contact' Sookootow referred to, but in the chaos the Unfettered would never know if something happened to the ships.`

Reviewer 2: ```suggestion
			`	The two mysterious black ships hail you, "Listen carefully. You may have noticed a Korath drone in the area, and they don't like Alpha dogs' presence in our space. If some Korath come to help, do not open fire on them, understood?" The other one then speaks, "We'll let you deal with that Bactrian, but I would recommend bribing 'neutral' pirates<, unfettered title>!" They then end the hail, making it clear they will not take no for an answer. They probably are allies of the 'contact' Sookootow referred to, but in the chaos the Unfettered would never know if something happened to the ships.`
```

		on kill
			"midas: wronged" ++
			set "midas: talon assassination"
			event "midas: thorn talon vengeance" 360
			dialog `You receive an ominous and final message of defiance, "Nemo me impune lacessit!"`
	# the Korath see this as an opportunity to hunt some Alpha dogs.
	npc
		government "Korath"
		personality surveillance
		ship "Kas'ik Tek 7"
	npc
		system "Kor Men"
		government "Korath (Military)"
		to despawn
			"unfettered: cleaned up cerberus" > 1
		personality entering disables plunders waiting heroic vindictive
		fleet "Korath Defense" 2
	npc disable
		government "Cerberus"
		personality plunders vindictive entering
		fleet "Small Northern Pirates" 4
		fleet "Large Northern Pirates" 2
		conversation "cerberus reinforcements"
	npc disable
		system "Alcyone"
		government "Cerberus"
		personality plunders vindictive waiting
		fleet "Large Northern Pirates" 3
		fleet "Small Northern Pirates" 2
		conversation "cerberus reinforcements"
	to fail
		not "<uhai> 7: Stolen Cargo: active"
	on fail
		"unfettered rep: Sookootow" --

mission "<uhai> 8: Unfettered Humans?"
	landing
	minor
	name "Learn more about the 'unfettered humans'?"

> 	description `The Auxiliary was manned by 'unfettered humans'. Find out what this means, pirates may have the answers.`

Reviewer 2: ```suggestion
	description `The Auxiliary was manned by 'unfettered humans'. Find out what this means. Pirates may have the answers.`
```

				die
			label hint
			`	Some of them already aim for you with remarkable speed, ready to shoot, but what you suspect to be their leader makes them lower their weapons again with a gesture of their hand. "The ones you seek are a common enemy, so I will tell you this: look far, human, and you may find them in the eye of a storm that has been raging for centuries." They then disappear as quickly as they had come.`

mission "<uhai> 9: Alpha Quest"
	name `Find the Auxiliary`
	description `Go to <destination> to learn where you can find the Alpha ship.`
	source "Darkcloak"
	destination "Firelode"
	to offer
		not "unfettered: destroyed auxiliary"
		has "<uhai> 7: Stolen Cargo: done"
		has "event: unfettered: analyzed the scans"
		has "unfettered: it's the Alphas!"
		has "Unfettered Jump Drive 3: offered"
	on offer
		conversation
			`As you enter the spaceport, you see the council gathered in a stadium looking similar to the one on Firelode, although it is larger. It seems like you will finally be able to get some answers, if you would like to see them now, that is.`
			choice
				`	(Enter the stadium and go see them.)`
				`	(I don't have time right now.)`
					defer

> 			`	Seeing you approach, one of the leaders steps forward, and you recognize Sookootow. "It seems you are an enemy of the unfettered humans. We cannot risk a disruption in our supply of jump drives. You may have provided some to us, but so did they," He says, speaking almost theatrically, with big gestures, "We have a task to ask of you. Should you accept, you would be able to join us - still not officially for now, but all good things come in time." You assume that by unfettered humans he means the Alphas.`

Reviewer 2: ```suggestion
			`	Seeing you approach, one of the leaders steps forward, and you recognize Sookootow. "It seems you are an enemy of the unfettered humans. We cannot risk a disruption in our supply of jump drives. You may have provided some to us, but so did they," he says, speaking almost theatrically, with big gestures, "We have a task to ask of you. Should you accept, you would be able to join us - still not officially for now, but all good things come in time." You assume that by unfettered humans he means the Alphas.`
```

			choice
				`	"How can I find them?"`
				`	"I already know the way."`
					to display
						has "alpha: host found"
					goto know
			`	He seems amused. "That, <first>, is the easy part. You still have those sensor logs about their ships... Maybe it can point you in a direction. You should probably go to Mera and ask her if she found anything useful in the logs."`
			choice
				`	"I better get on my way to kill Alphas then."`
				`	"I won't risk engaging the Alphas. This is over."`
					decline
			label warning
			`	Before you go, one of the warlords comes to speak to you personally. "Do not dare come back to us asking for more cooperation, should you fail this. That said, I'm not sure you would ever come back if defeated. But never forget that jump drives matter more than anything else. You should steal as many as you can if you do indeed manage to beat the Alphas."`
				accept
			label know

> 			`	You explain to him you have already found an Alpha base in the system of Host. "How can you know this is where you will find this Auxiliary? They could have went anywhere. I agree that it is the most likely spot, but you should still go listen to what Mera has to say, she may reveal tactical opportunities for you to exploit."`

Reviewer 2: ```suggestion
			`	You explain to him you have already found an Alpha base in the system of Host. "How can you know this is where you will find this Auxiliary? They could have gone anywhere. I agree that it is the most likely spot, but you should still go listen to what Mera has to say, for she may reveal tactical opportunities for you to exploit."`
```

				goto warning
	on enter
		dialog `You receive an ominous message from an unknown source. "I would recommend you stop working with the Unfettered, <last>."`
		log "Received a weird message demanding to stop working with the Unfettered. Could it have come from the Hai? The Alphas? Other enemies of the Unfettered?"

mission "<uhai> 9: Capture The Alpha Auxiliary"
	landing

> 	name `Find and capture the Alpha Auxiliary`

Reviewer 2: Is this too long as well?

					goto logs
				`	"What can we use that information for?"`
					goto information
			label logs
			`	"Yes, I noticed on the scanner logs. I would advise not relying too heavily on your shields, but they don't seem to have that many of those weapons, so you should be fine. I do not believe they could have designed such weapons by themselves. They probably just stole them from nearby aliens."`
			choice
				`	"What about the damage on their ship? How is that relevant to my mission?"`
					goto information
				`	"You should have asked me to do an outfit scan, earlier."`
					to display
						not "unfettered: scanned auxiliary's outfits"
				`	"I did well to scan its outfits anyway then, so we have some information to work with."`
					to display
						has "unfettered: scanned auxiliary's outfits"
					goto scanned
			`	"Maybe, but it could have provoked them further. They apparently like their secrets. I only wanted to know about the structure of the ship, but I'd agree that knowing more about those weapons would be useful. Now, back to the damage on their ship.`
				goto information
			label scanned
			action
				"unfettered rep: Mera" ++

> 			`	She moves her head to the side, surprised, "You... Did? Let me take a look at it then," she asks, and you give her the details. "It seems like this ship was outfitted for speed rather than combat, luckily for you. They are using 2 Tracker Pods, 2 Bullfrogs and one combo of turrets that disrupt shields and slice through hull. I am not much of a strategist so I'll let you decide what to make out of this intel," she says, before taking another look at the logs. And almost jumping in place, "Wait! They have a Systems Core, and plenty of fuel. That must mean that the damage to their ship should have been repaired very quickly. The trail could run cold very quickly in that case, I'd advise you to head east as soon as you can." She stops for a second, unsure what to say, "No, that can't be. Anyway, back to the damage suffered to their ship."`

Reviewer 2: ```suggestion
			`	She moves her head to the side, surprised, "You... did? Let me take a look at it then," she asks, and you give her the details. "It seems like this ship was outfitted for speed rather than combat, luckily for you. They are using 2 Tracker Pods, 2 Bullfrogs and one combo of turrets that disrupt shields and slice through hull. I am not much of a strategist so I'll let you decide what to make out of this intel," she says, before taking another look at the logs. And almost jumping in place, "Wait! They have a Systems Core, and plenty of fuel. That must mean that the damage to their ship should have been repaired very quickly. The trail could run cold very quickly in that case. I'd advise you to head east as soon as you can." She stops for a second, unsure what to say, "No, that can't be. Anyway, back to the damage suffered to their ship."`
```

					goto information
				`	"What were you about to say?"`
			`	She looks down once more, "Be careful, <last>. Something is not right here," she says, not willing to reveal more.`
			label information
			`	Her ears move slightly, as she starts to explain technical things to you, "Obviously, this amount of damage has consequences. It looks like they've quickly patched it up, so it won't affect their hull resistance to weapon's fire. However, this is interesting," she says, pointing out particles on her pad you have not seen before. "Another captain found weird particles in that same system. We've never found traces of those particles before. I guess it could mean a shot got through and did some damage to one of the jump drives they are carrying - maybe even the one they use to jump. One thing is certain: these particles are not left by normal human ships, or Hai for that matter. My best guess would be that when the ship jumped, the hole it left behind did not close as quickly as it should, and antimatter came from the other side. Perhaps it left these peculiar particles when being annihilated by contacting matter. Or maybe the particles just came through the opening in space?"`

> 			`	She shakes her head, getting focused again. "It doesn't matter much anyway, I must be boring you with this stuff," she says with a chuckle. "I suspect that they would have gone back to their base to at least take a look at it, so there should be traces leading to it. Something bothers me about this: from our experience, jump drives either work or explode. Maybe it's because the jump drive is not actually being used, and is only leaking particles passively, that this is possible? Or maybe the damage done by these weapons is really special. After all, they are more advanced than what we have, or what the aliens up north possess. Only the Quarg and Korath have a similar level of technology, and those are the races that are rumoured to have invented the jump drives, so it could be some of these weapons are intended to disrupt it specifically."`

Reviewer 2: ```suggestion
			`	She shakes her head, getting focused again. "It doesn't matter much anyway, and I must be boring you with this stuff," she says with a chuckle. "I suspect that they would have gone back to their base to at least take a look at it, so there should be traces leading to it. Something bothers me about this: from our experience, jump drives either work or explode. Maybe it's because the jump drive is not actually being used, and is only leaking particles passively, that this is possible? Or maybe the damage done by these weapons is really special. After all, they are more advanced than what we have, or what the aliens up north possess. Only the Quarg and Korath have a similar level of technology, and those are the races that are rumoured to have invented the jump drives, so it could be some of these weapons are intended to disrupt it specifically."`
```

	on enter "Celeborim"
		dialog `You are on the right track. There are small traces here as well.`
	on enter "Asikafarnut"
		dialog `You are still on the right path, and you even find some small debris of ships. They probably had to stay here a bit longer to fight them off.`
	on enter "Makferuti" "Sobarati"
		dialog `There are no traces here. They probably went to another system, looking for refuge from the hostile drones.`
	on enter "Silikatakfar" "Eneremprukt"
		dialog `You cannot find any traces here. This is not the right direction.`
	on enter "Mesuket"
		dialog `This system looks dangerous. There is too much interference from the debris to pick up the traces, but you figure they would try to avoid this system.`
	on enter "Persitar"
		dialog `This doesn't look like a good hiding spot...`
	on enter "Ferukistek"
		dialog `There are traces here indeed. You spot one system with no hyperspace links north-east of here. It seems like a logical place for the Alphas to go, considering the drones do not appear to have jump drives.`
	on enter "Host"
		dialog

> 			`You arrive in the system expecting to find a vault and the ship repairing, but it looks more like... It's a trap!`

Reviewer 2: ```suggestion
			`You arrive in the system expecting to find a vault and the ship repairing, but it looks more like... it's a trap!`
```

			`	There was no doubt who was in command of this vessel, however. The jump drives don't seem to be aboard the ship. You should head to the Alpha base to see if you can find them or anything else of interest.`
	npc
		government "Cerberus"
		system "Host"
		personality heroic nemesis staying
		ship "Far Osk 27" "Apathy"
		ship "Far Osk 27" "Vanity"
		ship "Far Osk 27" "Despair"
		ship "Far Osk 27" "Envy"
		ship "Far Osk 27" "Lust"
		ship "Far Osk 27" "Pride"
	# additional fighters if the Alphas were uncontested by Midas. one can use these to fill their Auxiliary with fighters
	npc
		government "Cerberus"
		system "Host"
		personality heroic nemesis staying
		to spawn
			has "midas: wronged"
		ship "Far Osk 27" "Wrath"
		ship "Far Osk 27" "Gluttony"
		ship "Far Osk 27" "Greed"
		ship "Far Osk 27" "Desecration"
	to fail
		has "Wanderers: Defend Vara Ke'sok: done"
	to fail
		has "unfettered: outcast"
	on fail
		dialog phrase "Essential Unfettered Mission Fail"
	on visit

> 		dialog `You have landed, but the Auxiliary is not your flagship, or you landed without the jump drives. Go and look for traces of fuel from the Auxiliary in systems neighboring Hi Yahr. Don't forget to land on the Alpha base to recover any information or jump drives it may have! Don't forget the Jump Drives need to be on your flagship or in cargo.`

Reviewer 2: I feel another capitalization debate coming here…

Author: uh I used JD uncapitalized and then capitalized
I wish I knew when to use which haha

Reviewer 2: So do I. If someone would establish a general guideline that would be very helpful. **sigh** @tibetiroka ?

Author: I think there was a PR to standardize it

Reviewer 3: It should be uncapitalized.

			label stadium
			`	The crowd leads you to the stadium.`
			`	The council seems surprised to see you return, especially the Unfettered warlord that annoyed you earlier, but they address you with a deeper respect than before. "You have proven yourself. Now we have almost enough jump drives for the next phase..." You look at them intensely, and after a few moments they realize, "Oh right, your payment! Here it is: <payment>! From now on, you will be referred to as Herald."`
			choice
				`	"I have found Korath prisoners on the Alpha ship."`
			`	This seems to trouble them deeply. They are split between those who want to interrogate the Korath for their technological secrets, and those who still remember too freshly the horrors of the Korath raids. The Chiefs are now the most active ones - some show scars or speak of lost family, others insult the Korath. A few, however, keep their calm.`
			`	Then, finally, one of them speaks up. He had remained quiet through the argument, probably thinking. You see many scars on his arms as he stands, and can only guess at what he has been through. He speaks calmly, yet firmly, strongly enough to make everyone listen.`

> 			`	You recognize Choot'k, he looks different without his armor but there can be no doubts that it's him.`

Reviewer 2: ```suggestion
			`	You recognize Choot'k. He looks much different without his armor but there can be no doubt that it's him.`
```

Author: much different, is that correct?

Reviewer 2: I think? If there’s a better synonym, use that.

Reviewer 2: Resolved in 3800bed

				to display
					has "Hai Reveal [B03] Contact Giti: done"

> 			`	"We have lost much, so much. Too much... But so have they. They are victims of that war as much as we are. The raids happened long ago, I didn't fight these Korath, I fought their ancestors. If we view it as fair to raid the Wanderers today, we must see what the Korath did in the same way.`

Reviewer 2: ```suggestion
			`	"We have lost much, so much. Too much... but so have they. They are victims of that war as much as we are. The raids happened long ago. I didn't fight these Korath, I fought their ancestors. If we view it as fair to raid the Wanderers today, we must see what the Korath did in the same way.`
```

			`	Surprisingly, the doors of the hangar open to let you in. Maybe it is an automated response to this ship, or whoever is down here assumed the Alphas won the fight. That said, if there is anyone here, it's probably a skeleton crew.`
			`	You and your crew carefully exit your ship, but you see no signs of life here. They could be hidden, so you proceed cautiously, looking for the jump drive vault. On the way you hear sounds - sounds that definitely did not originate from humans. Heading that way, you find a brig full of prisoners, with a lizard head displayed above the cells. They seem to have been mistreated by the Alphas, and you find what appears to be a crudely rigged translation device next to their cell, with information regarding these Korath, an alien species located north-east of human space that sometimes engages in raiding. They apparently used to control the territories nearby that are now owned by automata.`

> 			`	The captives themselves must have endured a lot, you should take them to Darkcloak and ask the Chiefs for help. You notice one of them has a very serious injury to their leg, it seems that it has been partially replaced by a crude prosthetic. He may not survive the transport.`

Reviewer 2: ```suggestion
			`	The captives themselves must have endured a lot, so you should take them to Darkcloak and ask the Chiefs for help. You notice one of them has a very serious injury to his leg, and it seems that it has been partially replaced by a crude prosthetic. He may not survive the transport.`
```

Reviewer 2: We’ll later it says He.

Author: ~~right~~ well congratulations, its a boy

			`	There are also multiple flight logs. It seems they had been using this Auxiliary as a mobile base and for bringing jump drives to the Unfettered. These logs probably contain indications of where they have been getting the jump drives, but analyzing all of them will take more time. It doesn't take much, though, to notice they have spent significant time near Korath Exile territory. It is now time to leave the planet and go see the Chiefs.`
			label warning
			`	"Before leaving, you remember that the Unfettered want you to show them the Auxiliary, and they probably expect you to land with it as flagship. Make sure you bring all ten jump drives back in your cargo or on the flagship, and that you keep this ship alive at all cost. It may be wise to wait a bit and replenish your fuel in this system."`
			# She will warn you if she likes you.
			branch accept

> 				has "unfettered rep: Mera"

Reviewer 2: What if it’s negative, does this still happen?

		"unfettered: cleaned up cerberus" ++
	`Now that all pirate reinforcements have been taken care of, and it is time to board the flagship and take their loot.`
		to display
			"unfettered: cleaned up cerberus" == 2
		decline
	`Part of the pirate reinforcements have been taken care of, but more remain.`
		to display
			"unfettered: cleaned up cerberus" == 1
		decline
	label boardedship
	branch noduel
		not "midas: accepted duel"
	`All pirate reinforcements have been taken care of, and the Sea Dragons signal you they are ready to start the duel whenever you will take off again.`
	label noduel
	`All pirate reinforcements have been taken care of, all that is left to do is to bring back the cargo.`

mission "<uhai> 7: Cavalry Arrives"
	landing
	invisible
	source "New Tortuga"
	to offer
		has "<uhai> 7: Stolen Cargo: active"
	on offer
		conversation
			`Realizing you have landed after them, the pirates fire up their engines and take off as fast as they can. They might not have expected you to have the guts to land on such a lawless world. Or maybe they just wanted to delay you... Whatever the case, you quickly order your crew to get back into space.`
				launch
	on enter
		conversation

> 			"As you enter the system it becomes clear: Cerberus have called for backup! Multiple pirate ships are arriving in the system. But you also see other ships arriving by jump drives. They do not appear to be pirates, maybe the 'contact' has provided help as well?"

Reviewer 2: ```suggestion
			`As you enter the system it becomes clear: Cerberus has called for backup! Multiple pirate ships are arriving in the system. But you also see other ships arriving by jump drives. They do not appear to be pirates. Maybe the 'contact' has provided help as well?`
```

		ship "Sea Dragon" "Thorn"
		on provoke
			dialog `The ships both hail you, "What exactly do you think you are doing, <first> <last>? Nemo Me Impune Laccessit. You do not want to know what it means."`
		on kill
			"midas: wronged" ++
			set "midas: thorn assassination"
			event "midas: thorn talon vengeance" 360
			dialog `You receive an ominous and final message of defiance, "Nemo me impune lacessit!"`
	npc save
		government "Shadow"
		personality entering heroic uninterested staying mute
		ship "Sea Dragon" "Talon"
		on provoke
			dialog `The ships both hail you, "What exactly do you think you are doing, <first> <last>? Nemo Me Impune Laccessit. You do not want to know what it means."`
		on kill
			"midas: wronged" ++
			set "midas: talon assassination"
			event "midas: thorn talon vengeance" 360
			dialog `You receive an ominous and final message of defiance, "Nemo me impune lacessit!"`
	# the Korath see this as an opportunity to hunt some Alpha dogs.
	npc
		government "Korath"
		personality surveillance
		ship "Kas'ik Tek 7"
	npc
		system "Kor Men"
		government "Korath (Military)"
		to despawn
			"unfettered: cleaned up cerberus" > 1

> 		personality entering disables plunders waiting heroic vindictive

Reviewer 2: Aren't `disables` and `vindictive` opposites?

		fleet "Korath Defense" 2
	npc disable
		government "Cerberus"
		personality plunders vindictive entering
		fleet "Small Northern Pirates" 4
		fleet "Large Northern Pirates" 2
		conversation "cerberus reinforcements"
	npc disable
		system "Alcyone"
		government "Cerberus"
		personality plunders vindictive waiting
		fleet "Large Northern Pirates" 3
		fleet "Small Northern Pirates" 2
		conversation "cerberus reinforcements"
	to fail
		not "<uhai> 7: Stolen Cargo: active"
	on fail
		"unfettered rep: Sookootow" --

mission "<uhai> 8: Unfettered Humans?"
	landing
	minor
	name "Learn more about the 'unfettered humans'?"
	description `The Auxiliary was manned by 'unfettered humans'. Find out what this means. Pirates may have the answers.`

> 	destination "Darkcloak"

Reviewer 2: No source?

		"actually a Syndicate advertisement"
		"a plot by the Alphas" 2
		"a ship from the Mercenaries" 2
		"a Cerberus vessel" 3
		"on a secret Navy mission" 2
		"a Free Worlds terrorist group"
		"part of Deep Security"
	word
		" and offers more information for a large sum of credits."
		" but they can provide more information... for a price."
		" but someone in a dimly-lit alley can tell you more." 2
		" if you would only follow them into a secret place, you could get more information."
		" and only a large sum of money will reveal details."
		" but a few credits will find the full truth."
	word
		" You quickly realize you are being scammed."

# if you betrayed Midas, you will suffer to find answers
mission "<uhai> 8: Unfettered Humans? Hint"
	landing
	invisible
	to offer
		has "<uhai> 5: Scan Auxiliary: done"
		not "midas: wronged"
	deadline 15
	on enter "Misam"
		fail
	to fail
		has "unfettered: it's the Alphas!"
	on fail
		conversation
			`Once again you hear a voice when you are alone in your quarters.`
			`	"Come to us if you truly want to know who that Auxiliary belongs to." The message comes with a location on New Tortuga.`
				to display
					not "unfettered: it's the Alphas!"

> 			`	"Congratulations. It seemed that you did not need our help to find these Alphas."`

Reviewer 2: ```suggestion
			`	"Congratulations. It seems that you did not need our help to find these Alphas."`
```

			`	You order another drink and look at the mural in more detail. You nearly choke on your drink when you realize that the exposed face and hands of the knight are green.`
				decline
			label "dead, boy!"
			`As soon as you set foot on the planet, a team of mercenaries comes out of the shadows, equipped with the same kind of armor Si-Ra had. Their weapons may be holstered, but you can feel the tension in the air. Their leader speaks up. "We know your query. Consider yourself lucky to not be killed on the spot. We do not guarantee that will not happen, should you land again."`
			`	They evaluate their surroundings before coming closer to you, and once they are a meter away they stop, clearly speaking their words as a threat, with a pause between each of them "Nemo. Me. Impune. Laccesit."`
			choice
				`	"Surely there is something you can tell me?"`
					goto hint

> 				`	(Threaten them).`

Reviewer 2: ```suggestion
				`	(Threaten them.)`
```

> 				`	(Leave).`

Reviewer 2: ```suggestion
				`	(Leave.)`
```

			choice
				`	(Enter the stadium and go see them.)`
				`	(I don't have time right now.)`
					defer
			`	Seeing you approach, one of the leaders steps forward, and you recognize Sookootow. "It seems you are an enemy of the unfettered humans. We cannot risk a disruption in our supply of jump drives. You may have provided some to us, but so did they," he says, speaking almost theatrically, with big gestures, "We have a task to ask of you. Should you accept, you would be able to join us - still not officially for now, but all good things come in time." You assume that by unfettered humans he means the Alphas.`
			choice
				`	"What is it you want me to do? I don't like these Alphas."`
					goto hate
				`	"Who supplies your jump drives is none of my concern."`
			`	"Good, and neither should it be, but if you want to join us, you will have to prove yourself. Beat that Auxiliary and bring back any loot you may find, we will reward you handsomely for it."`
				goto next
			label hate
			`	"Perfect! We want you to prove your worth. Beat that 'Alpha' Auxiliary and return here for the greatest reward you can imagine."`
			label next

> 			`	"And while you're at it, find out where they are getting these jump drives from. You can become our new supplier for even richer rewards. Of course, if you cannot obtain such information, we will at least ask for proof that you have taken on these 'Alphas', by bringing their ship back."`

Reviewer 2: ```suggestion
			`	Sookootow continues, "And while you're at it, find out where they are getting these jump drives from. You can become our new supplier for even richer rewards. Of course, if you cannot obtain such information, we will at least ask for proof that you have taken on these 'Alphas', by bringing their ship back."`
```

			choice
				`	"What about the damage on their ship? How is that relevant to my mission?"`
					goto information
				`	"Can you tell me more about what kind of damage it was, on their ship?"`
			label damage
			`	"The damage was done by some kind of beam weapon, and a very strong one at that. It looks like it only got partially deflected by the shields, if at all. I wonder what kind of weapon can do that." She looks intrigued, and pauses to think about it, then continues, "That would explain why it's forbidden for us to venture east with jump drives. We lack any hull repair capabilities, and to be honest with you, our hull is not much more resilient than the one of some human ships, like that Auxiliary."`
			choice
				`	"They seemed to be using the same weapon against me."`

> 					goto logs

Reviewer 2: ```suggestion
```

				`	"What can we use that information for?"`
					goto information
			label logs
			`	"Yes, I noticed on the scanner logs. I would advise not relying too heavily on your shields, but they don't seem to have that many of those weapons, so you should be fine. I do not believe they could have designed such weapons by themselves. They probably just stole them from nearby aliens."`
			choice
				`	"What about the damage on their ship? How is that relevant to my mission?"`
					goto information
				`	"You should have asked me to do an outfit scan, earlier."`
					to display
						not "unfettered: scanned auxiliary's outfits"
				`	"I did well to scan its outfits anyway then, so we have some information to work with."`
					to display
						has "unfettered: scanned auxiliary's outfits"
					goto scanned

> 			`	"Maybe, but it could have provoked them further. They apparently like their secrets. I only wanted to know about the structure of the ship, but I'd agree that knowing more about those weapons would be useful. Now, back to the damage on their ship.`

Reviewer 2: ```suggestion
			`	"Maybe, but it could have provoked them further. They apparently like their secrets. I only wanted to know about the structure of the ship, but I'd agree that knowing more about those weapons would be useful. Now, back to the damage on their ship."`
```

					goto information
				`	"What were you about to say?"`
			`	She looks down once more, "Be careful, <last>. Something is not right here," she says, not willing to reveal more.`
			label information
			`	Her ears move slightly, as she starts to explain technical things to you, "Obviously, this amount of damage has consequences. It looks like they've quickly patched it up, so it won't affect their hull resistance to weapon's fire. However, this is interesting," she says, pointing out particles on her pad you have not seen before. "Another captain found weird particles in that same system. We've never found traces of those particles before. I guess it could mean a shot got through and did some damage to one of the jump drives they are carrying - maybe even the one they use to jump. One thing is certain: these particles are not left by normal human ships, or Hai for that matter. My best guess would be that when the ship jumped, the hole it left behind did not close as quickly as it should, and antimatter came from the other side. Perhaps it left these peculiar particles when being annihilated by contacting matter. Or maybe the particles just came through the opening in space?"`

> 			`	She shakes her head, getting focused again. "It doesn't matter much anyway, and I must be boring you with this stuff," she says with a chuckle. "I suspect that they would have gone back to their base to at least take a look at it, so there should be traces leading to it. Something bothers me about this: from our experience, jump drives either work or explode. Maybe it's because the jump drive is not actually being used, and is only leaking particles passively, that this is possible? Or maybe the damage done by these weapons is really special. After all, they are more advanced than what we have, or what the aliens up north possess. Only the Quarg and Korath have a similar level of technology, and those are the races that are rumoured to have invented the jump drives, so it could be some of these weapons are intended to disrupt it specifically."`

Reviewer 2: ```suggestion
			`	She shakes her head, getting focused again. "It doesn't matter much anyway, and I must be boring you with this stuff," she says with a chuckle. "I suspect that they would have gone back to their base to at least take a look at it, so there should be traces leading to it. Something bothers me about this: from our experience, jump drives either work or explode. Maybe it's because the jump drive is not actually being used, and is only leaking particles passively, that this is possible? Or maybe the damage done by these weapons is really special. After all, they are more advanced than what we have, or what the aliens up north possess. Only the Quarg and Korath have a similar level of technology, and those are the races that are rumored to have invented the jump drives, so it could be some of these weapons are intended to disrupt it specifically."`
```

			label agree
			`	You were ready to go, but she seems to have something more to say. "Oh, and one last thing before you leave: the base is probably far away, so I would advise taking some fuel pods.`
			branch nogift
				"unfettered rep: Mera" < 2

> 			`	"In fact, here is a gift for you to help with that: a Fuel Processor. Don't ask how I got it and you can keep it, deal?`

Reviewer 2: ```suggestion
			`	"In fact, here is a gift for you to help with that: a Fuel Processor. Don't ask how I got it and you can keep it, deal?"`
```

			`	You find a few squirrel symbols, and a fist that could indicate they were coerced. The Alphas seem to often communicate using simple symbols for their logs as a primitive code, on top of the more serious encryptions used for sensitive information. Going deeper into these recent logs of activities, you find a monkey symbol, with an eye next to it. But it is not all, there is a symbol of a crossed portal, with a red skull icon.`
			`	There are also multiple flight logs. It seems they had been using this Auxiliary as a mobile base and for bringing jump drives to the Unfettered. These logs probably contain indications of where they have been getting the jump drives, but analyzing all of them will take more time. It doesn't take much, though, to notice they have spent significant time near Korath Exile territory. It is now time to leave the planet and go see the Chiefs.`
			label warning

> 			`	"Before leaving, you remember that the Unfettered want you to show them the Auxiliary, and they probably expect you to land with it as flagship. Make sure you bring all ten jump drives back in your cargo or on the flagship, and that you keep this ship alive at all cost. It may be wise to wait a bit and replenish your fuel in this system."`

Reviewer 2: ```suggestion
			`	Before leaving, you remember that the Unfettered want you to show them the Auxiliary, and they probably expect you to land with it as flagship. Make sure you bring all ten jump drives back in your cargo or on the flagship, and that you keep this ship alive at all cost. It may be wise to wait a bit and replenish your fuel in this system.`
```

	explode "large explosion" 40
	explode "huge explosion" 10
	"final explode" "final explosion large"

> 	description "This is a stolen Auxiliary that was used by the Alphas to move supplies and outfits around, mainly for trading with the Unfettered. It seems that did not modify it much and conveniently retained their logs, although in an encrypted form."

Reviewer: _Who_ did not modify it much?

			choice
				`	"I will land, as you ask."`
					accept
				`	"I will land, once I'm done plundering it."`
					to display

> 						has "disabled auxiliary"

Author: Idk, you could have already plundered it for one, and you might not want to provoke the uhai too

	cargo
		outfits
			"Building Materials" 142
		commodities

> 			"Equipement" 100

Reviewer 3: ```suggestion
			"Equipment" 100
```

	on enter
		conversation
			"As you enter the system it becomes clear: they have called for backup! Multiple pirate ships are arriving in the system. But you also see other ships arriving by jump drives. They do not appear to be pirates, maybe the 'contact' has provided help as well?"

> 			`They hail you, "Listen carefully. You may have noticed a Korath drone in the area, and they don't like Alpha dogs around much. If some Korath come to help, do not open fire on them, understood?" "We'll let you deal with that bactrian, but I would recommend you make sure these pirate ships are on your side<, unfettered title>!" The other ship adds, before ending the hail, making it clear they will not take no for an answer.`

Reviewer: ```suggestion
			`They hail you, "Listen carefully. You may have noticed a Korath drone in the area, and they don't like Alpha dogs around much. If some Korath come to help, do not open fire on them, understood? We'll let you deal with that bactrian, but I would recommend you make sure these pirate ships are on your side<, unfettered title>!" The other ship adds, before ending the hail, making it clear they will not take no for an answer.`
```

mission "<uhai> 8: Unfettered Humans?"
	landing
	minor
	name "Learn more about the 'unfettered humans'?"
	description `The Auxiliary was manned by 'unfettered humans'. Find out what this means, pirates may have the answers.`
	destination "Darkcloak"
	to offer
		has "<uhai> 5: Auxiliary Scans: done"
	on offer
		conversation

> 			`You learned from the Unfettered Hai that the Auxiliary you fought recently in order to get the scans was in fact manned by 'unfettered humans'. It would be wise to find out what that means, and where they come from, especially considering they shot you on sight.`

Reviewer: ```suggestion
			`You learned from the Unfettered Hai that the Auxiliary you fought recently in order to get the scans was in fact manned by 'unfettered humans'. It would be wise to find out what that means, and where they come from, especially considering they shot at you on sight.`
```
